ci/sanity/install.sh

```diff
@@ -34,7 +34,7 @@ kubectl apply -f ./deploy/cstor-operator.yaml
 
 echo "Install CSI"
 
-kubectl apply -f ./deploy/csi-operator.yaml
+kubectl apply -f ./deploy/csi-operator-ubuntu-18.04.yaml
```

Comment:
@mittachaitu we have to update the quick.md with new filename ^^ and some details about the OS specific mounts info , we can take it from the cstor-csi repo README

Reply:
Vani is doing that, for more info https://kubernetes.slack.com/archives/C015124PVTL/p1594806518491600?thread_ts=1594731687.459600&cid=C015124PVTL